text/0030-standardized-sender-sig.md

```diff
@@ -0,0 +1,137 @@
+- Feature Name: Standardized Sender Signature
+- Start Date: 2022-03-14
+- MCIP PR: [mobilecoinfoundation/mcips#0030](https://github.com/mobilecoinfoundation/mcips/pull/0030)
+- MobileCoin Epic: None
+
+# Summary
+[summary]: #summary
+
+Enact MCIP #18 (Janus Mitigation), which sets the ephemeral key in the encrypted fog hint
+to be the tx private key times the Ristretto base point. The original motivation was to enact
+Janus attack mitigation.
+
+We find a secondary use-case to motivate this: it becomes possible for the signer to create
+a Schnorr signature over any payload, which a verifier can validate against the keys appearing
+in a `TxOut`, without revealing the identity of the signer. We propose to both do #18 and standardize
+the construction and validation of this signature.
+
+# Motivation
+[motivation]: #motivation
+
+Often in a payments flow, it is necessary for the sender and recipient to exchange additional messages
+and metadata about the payment. Frequently, these messages need to be authenticated.
+
+We have a mechanism for doing this in the memo field, which attaches fixed-size payloads to TxOut's.
+However, this imposes fairly strict size requirements on the data that can be sent.
+
+Outside of the memo field, we have a notion of "receipts" and "confirmation numbers" which a sender
+can use to identify themselves to a recipient. These are hashes of the TxOut shared secret, which
+are similarly deniable, and convince the recipient that whoever sent them this number must have access
+to secrets involved in the construction of the TxOut.
+
+However, these numbers are useless for convincing a third-party who is not a party to the transaction,
+because only the recipient is able to validate these numbers. There is currently no form of receipt that
+can be validated just from the blockchain.
+
+This is desirable by some app project developers, who would like to store encrypted metadata per TxOut,
+but off-chain. For example, they might like to have a database of encrypted metadata, keyed on TxOut's sent
+by users of their app, and be able to ensure that only the true sender of the TxOut can post metadata associated
+to the TxOut, without learning anything about the sender, recipient, amount, etc. of the TxOut.
+
+After this proposal, a straightforward way to do this is that their app can sign the encrypted metadata using
+the tx private key, and their server can check this signature against the public key in the encrypted fog hint.
+We can simply use the Schnorrkel-based signature with Ristretto curve points that we already use in fog public addresses.
+
+This might also be useful because it makes the notion of receipts somewhat more robust -- arbitrary metadata
+can be signed by the sender, and this form of signature will not have the deniability property.
```

Comment:
Can you say more about why it does not have the deniability property?

Reply:
The receipts based on the shared secret, have the deniability property, because both the sender and recipient can build the shared secret, and so form the signature. So it is technically possible for the recipient to have forged the signature, and for the sender to deny having created the receipt.

With this signature, the sender is signing it with the tx private key, which is a key only they have access to, and the recipient can't fake the signature using the view private key. So this signature is not deniable, it must have come from the sender.

Reply:
Awesome - it may be worth including some of the text in your answer to define what we mean be deniability property